projects/ngrx.io/content/guide/component-store/comparison.md

```diff
@@ -1,3 +1,12 @@
+<div class="alert is-helpful">
+
+**<a href="/guide/signals"><b>NgRx Signals</b></a> is the new default.**
+
+The NgRx team recommends using `@ngrx/signals` library for local state management in Angular.
```

Comment:
```suggestion
The NgRx team recommends using the `@ngrx/signals` library for local state management in Angular.
```

Reply:
Should we encourage migrating existing ComponentStores to SignalStore?

Reply:
Yes… and just an idea: I would even suggest adding a note that for new projects, `@ngrx/signals` should be the recommended choice for global state management. Many applications out there are not so complicated to require Redux for global management.